src/App.js

```diff
@@ -43,7 +54,7 @@ function App() {
   return (
     <div className="App">
       <section>
-        {/* if the player names are numbers, that will make our life easier later because we can reuse numbers as arrays. Note that this will make our app brittle! */}
+       
         <Player
           to={to}
```

Comment:
You shouldn't pass the pieces of state you set in Context through prop drilling. You've kept prop drilling in several components, but since you properly set up your Context, you can grab the state you need in each file from Context.